pkg/input/wordlist.go

```diff
@@ -117,13 +118,23 @@ func (w *WordlistInput) readFile(path string) error {
 					data = append(data, []byte(contnt))
 				}
 			} else {
-				data = append(data, []byte(reader.Text()))
+				text := reader.Text()
+
+				if w.config.IgnoreWordlistComments && strings.HasPrefix(text, "#") {
```

Comment:
If we want to remove all the comments for entries like:
```
# this will be removed
 # this had an extra space in the beginning, and won't
valid_entry # this is a comment for the valid entry
```

The current code will  only catch the first one.